theano/sandbox/cuda/basic_ops.py

```diff
@@ -3300,23 +3282,20 @@ def make_node(self, value, *shape):
                 const_shp = None
             bcast.append(numpy.all(1 == const_shp))
         otype = CudaNdarrayType(dtype='float32', broadcastable=bcast)
-        return Apply(self, [v] + sh, [otype()])
+        return Apply(self, sh, [otype()])
```

Comment:
I've put it in `theano.tensor.type` along with the other `values_eq_approx` variants, named `values_eq_approx_always_true`.
And `values_eq_approx` is an attribute of the type, not of the variable, so I'm just setting `otype.values_eq_approx` instead of instantiating `otype()` first (note that `otype().type is otype`).